src/services/items/fluent-schema.ts

```diff
@@ -42,7 +42,7 @@ const item = S.object()
 // type 'base' (empty extra {})
 const baseItemCreate = S.object()
   .additionalProperties(false)
-  .prop('name', S.string().minLength(1).pattern('^\\S+( \\S+)*$'))
+  .prop('name', S.string().minLength(1).pattern('^[ \\S]+$'))
```

Comment:
I think this is not the regex suggested in the related issue: `^\S+([ +\S⁄])*$`

The current one allows a single space as a name.